script/sheet/actor.js

```diff
@@ -78,14 +78,35 @@ export class DarkHeresySheet extends ActorSheet {
     await prepareCommonRoll(rollData);
   }
 
+  _getCharacteristicOptions (selected) {
+    const characteristics = []
+    for (let char of Object.values(this.actor.data.data.characteristics)) {
+      characteristics.push({
+        label: char.label,
+        base: char.base,
+        selected: char.short === selected
+      })
+    }
+    return characteristics
+  }
+
   async _prepareRollSkill(event) {
     event.preventDefault();
     const skillName = $(event.currentTarget).data("skill");
     const skill = this.actor.data.data.skills[skillName];
+    const defaultChar = skill.defaultCharacteristic || skill.characteristics[0]
```

Comment:
Where is `skill.defaultCharacteristic` defined?

Reply:
Oh good catch! I had initially thought to give each skill a default, but then I found in `template.json` that each skill sort of already has that in its `characteristics` array. 

So to answer your question, it isn't currently defined anywhere, but if a skill were to want a different default than the first item in `characteristics` it would be defined in `template.json`

Reply:
Ah gotcha. 

You can assume that `skill.characteristics[0]` is the "base characteristic" (Alternate Characteristics, CB pg96) and should be selected by default, and the remaining elements in the array are suggested alternatives.

The `skill.characteristics` should always have at least one element, or it's bad data.